### DISCUSSION TITLE
DAEX-13 Mk5 contact pin

### DISCUSSION THREAD

**Discussion by bobincomox** [RChadwick7/FreedomWavePD#9] [Q&A]
Hello,

I am currently putting together your controller by following along with your really well done video. Unfortunately I am learning to solder as I do this so the process is slower than I originally thought. I do have enough components to make 2 boards plus spares, so I am hopeful I will get a good one in the end!

Due to the unavailability of the Tectonic TEAX09C005-8 I settled on the Dayton DAEX-13-4SM for the actuators, mainly because of the Parkgloveman DAEX 13 Mounting .stl files in this github. I looked at the thingiverse linked to by WinnieThePoo's description, and did not see the Mk5 contact there. Is there another thingiverse by WTP that shows where it fits in? Google searches for 'winnie' are flooded by the Pooh Bear...

My guess is the Mk5 contact floats on top of the adjustment screw providing a more focussed contact point that can move around slightly in the top plate?

Thanks so much for sharing your knowledge. I intend to post my progress here as progress is made. I live on Vancouver Island, Canada. My brother has PD. I am hoping to have something for him in September, 2024.

Cheers,
Bob

**Comment by RChadwick7** [comment]
Take your time! Especially if you're new to soldering, practice will help. A piece of advice I just thought of... Putting a piece of tape over the components on modules you're soldering (Such as the DRV2605Ls) can prevent accidentally putting solder blobs where they will cause problems.

The link to Thingiverse is the only public repository I know of. I was chatting with WTP, and he sent me files directly. You can contact him over at the Health Unlocked forum: https://healthunlocked.com/cure-parkinsons.

Unfortunately, I don't know much about the DAEX actuators. It's definitely on my to-do list and I have all the parts, just didn't have the time. When I get a chance to, I'll post on here what I come up with.

I hope these help your brother. Please let me know how they do. If you have any questions or issues, feel free to ask.

**Comment by bobincomox** [comment]
I finished soldering 1 board which at least had the LCD display light up and the Arduino LED's on when power was plugged into the USB port. I replaced the solder and flux I was using to 63/37 thin (23 gauge) solder and a syringe delivered flux paste. This change made it easier for me to be more precise while soldering. Note that the LCD contrast control was still off from the board's access hole, and the 4 pins were still a little off, but not nearly so much as in the video.

I discovered the Mk5 contact pin is from Winnie the Pooh's latest rendition of his 3-D printed mounting box for the DAEX-9-4SM exciter. The other .stl files are for the DAEX-13-4SM mounting box which is his Mk4 design.

Now for the software hurdles, which I will make another post for in due time...

Bob
